### PR TITLE
New package: Minder-1.0.8

### PR DESCRIPTION
**Pull request by CameronNemo**

### CODE REVIEW

File: srcpkgs/Minder/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'com.github.phase1geo.Minder'
+pkgname=Minder
+version=1.0.8
+revision=1
+build_style=meson
+hostmakedepends="git pkg-config"
+makedepends="vala-devel glib-devel cairo-devel granite-devel gtk+3-devel libxml2-devel libgee08-devel"
+depends=""
```
Comment (maxice8) [inline srcpkgs/Minder/template]: empty variable

File: srcpkgs/Minder/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'com.github.phase1geo.Minder'
```
Comment (maxice8) [inline srcpkgs/Minder/template]: Template file for 'Minder'

File: srcpkgs/Minder/template [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+pkgname=Minder
+version=1.0.8
+revision=1
+build_style=meson
+hostmakedepends="git pkg-config"
+makedepends="vala-devel glib-devel cairo-devel granite-devel gtk+3-devel libxml2-devel libgee08-devel"
+depends=""
+short_desc="Mind-mapping application designed for Pantheon"
+maintainer="Cameron Nemo <camerontnorman@gmail.com>"
+license="GPL-3.0-only"
```
Comment (maxice8) [inline srcpkgs/Minder/template]: GPL-3.0-or-later

Comment (CameronNemo) [reply srcpkgs/Minder/template]: huh looked like 3.0-only to me, how can I tell in the future?

**Comment by CameronNemo** [comment]
Addressed comments.

**Comment by maxice8** [comment]
On Sun, Aug 12, 2018 at 06:40:19PM -0700, Cameron Nemo wrote:
> CameronNemo commented on this pull request.
> 
> 
> 
> > @@ -0,0 +1,14 @@
> +# Template file for 'com.github.phase1geo.Minder'
> +pkgname=Minder
> +version=1.0.8
> +revision=1
> +build_style=meson
> +hostmakedepends="git pkg-config"
> +makedepends="vala-devel glib-devel cairo-devel granite-devel gtk+3-devel libxml2-devel libgee08-devel"
> +depends=""
> +short_desc="Mind-mapping application designed for Pantheon"
> +maintainer="Cameron Nemo <camerontnorman@gmail.com>"
> +license="GPL-3.0-only"
> 
> huh looked like 3.0-only to me, how can I tell in the future?
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/1704#discussion_r209477340

The Headers of the source code files most of the time, this time the work was already done with debian/copyright

**Pull request merged**